buildconfig/pygame-stubs/mixer.pyi

```diff
@@ -19,7 +19,7 @@ def init(
     buffer: int = 512,
     devicename: Optional[str] = None,
     allowedchanges: int = 5,
-) -> None: ...
+) -> True: ...
```

Comment:
This probably returns whether init was successful, not just True.

Also, if you're going to fix that you've got to fix the docs too, because they say it returns None.

Reply:
My fault, getting mixed up with which changes I'd made as I was looking at my fork on 2 different machines, think I committed the wrong one as the other one I'd changed to bool, not True. Also updated docs, apologies. I feel like a noob!